### PR TITLE
resolve issue with header stories throwing errors

### LINKED ISSUE #123: Certain stories in Header throws errors
### Description of issue:
https://nypl.github.io/nypl-design-system/storybook/storybook-static/react/index.html?path=/story/heading--headings-with-bold is throwing an error instead of displaying the story

### LINKED ISSUE #140: Move all components up to top level
Currently our components are hidden within `atom/`, `molecule/`, and `organism` folders. We'd like to bring all of these up to the top level to make the sidebar in Storybook more useful (alphabetical).

### LINKED ISSUE #141: Move tests into their corresponding component folders
We'd like to move tests into their corresponding component folders. Blocked by #140.

### PR DESCRIPTION
**Pull request by M0nica**
Resolve: https://github.com/NYPL/nypl-design-system/issues/123 

## **This PR does the following:**
- Add React.fragments around the bold and link header examples in order to prevent multiple child error from being thrown 

### Front End Review:
- [ ] View https://nypl.github.io/nypl-design-system/storybook/storybook-static/react/index.html?path=/story/heading--headings-with-bold
- [ ] View https://nypl.github.io/nypl-design-system/storybook/storybook-static/react/index.html?path=/story/heading--headings-with-link

### CODE REVIEW

**Review by helenvholmes — CHANGES_REQUESTED** [review]
Nice, so great to not have these throwing errors.

@M0nica, we have such a large number of components that we'd like to flatten @hokei and I talked about moving things up to `src/react-components/` as we touched any particular file (like you did for #137). Can you:

- Move `Heading` up to the top level as part of this PR and
- Update the package.json and CHANGELOG?

Thank you!

**Comment by M0nica** [comment]
> Nice, so great to not have these throwing errors.
> 
> @M0nica, we have such a large number of components that we'd like to flatten @hokei and I talked about moving things up to `src/react-components/` as we touched any particular file (like you did for #137). Can you:
> 
>     * Move `Heading` up to the top level as part of this PR and
> 
>     * Update the package.json and CHANGELOG?
> 
> 
> Thank you!

👍 I just updated to move the `Headings/` from `/02-molecules` into top-level `/components` and updated the CHANGELOG accordingly.

**Review by helenvholmes — APPROVED** [review]
Thanks @M0nica, LGTM!

File: src/react-components/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nypl/design-system-react-components",
-  "version": "0.3.6",
+  "version": "0.3.7",
```
Comment (hokei) [inline src/react-components/package.json:3]: A note:  There's no need to bump this every time, just after a release.  The last release was `0.3.4` so we should be at `0.3.5` (with an implied prerelease)

Comment (helenvholmes) [reply src/react-components/package.json:3]: Whoops, that was me requesting this. Sorry!

**Pull request merged**